Editor/LocalisationEditor.cs

```diff
@@ -107,15 +110,15 @@ private void OnGUI()
 				"Click Generate to build a class file which can be referenced application-side to get localisation keys and other data in-game.",
 				MessageType.Info);
 			EditorGUILayout.LabelField("Output filename:", CONFIG_FILENAME);
-			EditorGUILayout.LabelField("Localisation data path:", "Resources/" + LOC_DATA_PATH + "/");
+			EditorGUILayout.LabelField("Localisation data path:", "Resources/" + LOC_FOLDER_NAME + "/");
 
-			EditorGUILayout.LabelField("Template required to generate locale class");
+			EditorGUILayout.LabelField("Template required to generate localisation class");
 			EditorGUILayout.BeginHorizontal();
 			configTemplate =
 				(TextAsset)EditorGUILayout.ObjectField("Config template:", configTemplate, typeof(TextAsset), false);
 
 			EditorGUI.BeginDisabledGroup(configTemplate == null);
-			if (GUILayout.Button("Generate Locale Class"))
+			if (GUILayout.Button("Generate Loc Class"))
```

Comment:
What is a `Loc` class. I think the previous label was a little more descriptive.

Reply:
Renamed